packages/common/src/types/api/email.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Email
+ * @pattern ^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$
+ */
+export type Email = string;
```

Comment:
only thing I don't understand is how this pattern is consumed 🤔

Reply:
Is consumed by `tsoa` lib when we run `yarn workspace backend generate-api` which generates files that validate API requests and documentation. 

Docs on tsoa @pattern annotation https://tsoa-community.github.io/docs/annotations.html

Reply:
tyyy!!! makes sense